From 368703af1b3e54e8ed15a5897c9f8ba390288b80 Mon Sep 17 00:00:00 2001
From: Josua Mayer <josua@solid-run.com>
Date: Wed, 19 Apr 2023 14:45:02 +0300
Subject: [PATCH] lx2160acex7: half-twins: regenerate dpc with "compatible"
 properties

Apparently this is required for dpmac to be created now, in MC fw.
10.36.0 ...

Signed-off-by: Josua Mayer <josua@solid-run.com>
---
 config/lx2160a/CEX7/dpl-eth.8x10g.8x1g.dts | 75 +++++++++++++++++++++-
 1 file changed, 72 insertions(+), 3 deletions(-)

diff --git a/config/lx2160a/CEX7/dpl-eth.8x10g.8x1g.dts b/config/lx2160a/CEX7/dpl-eth.8x10g.8x1g.dts
index 6c859be..873fd36 100644
--- a/config/lx2160a/CEX7/dpl-eth.8x10g.8x1g.dts
+++ b/config/lx2160a/CEX7/dpl-eth.8x10g.8x1g.dts
@@ -13,14 +13,16 @@
 
 			objects {
 
-				/* ------------ DPMACs --------------*/
+				/* -------------- DPMACs --------------*/
 				obj_set@dpmac {
 					type = "dpmac";
-					ids = <0x3 0x4 0x5 0x6 0x7 0x8 0x9 0xa 0xb 0xc 0xd 0xe 0xf 0x10 0x11 0x12>;
+					ids = <3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 >;
 				};
+
+				/* -------------- DPMCPs --------------*/
 				obj_set@dpmcp {
 					type = "dpmcp";
-					ids = <0x1 0x2 0x3 0x4 0x5 0x6 0x7 0x8 0x9 0xa 0xb 0xc 0xd 0xe 0xf 0x10 0x11 0x12 0x13 0x14 0x15 0x16 0x17 0x18 0x19 0x1a 0x1b 0x1c 0x1d 0x1e 0x1f 0x20 0x21 0x22 0x23>;
+					ids = <1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 >;
 				};
 			};
 		};
@@ -32,140 +34,207 @@
 	objects {
 
 		dpmac@3 {
+			compatible = "fsl,dpmac";
 		};
+
 		dpmac@4 {
+			compatible = "fsl,dpmac";
 		};
+
 		dpmac@5 {
+			compatible = "fsl,dpmac";
 		};
+
 		dpmac@6 {
+			compatible = "fsl,dpmac";
 		};
+
 		dpmac@7 {
+			compatible = "fsl,dpmac";
 		};
+
 		dpmac@8 {
+			compatible = "fsl,dpmac";
 		};
+
 		dpmac@9 {
+			compatible = "fsl,dpmac";
 		};
+
 		dpmac@10 {
+			compatible = "fsl,dpmac";
 		};
+
 		dpmac@11 {
+			compatible = "fsl,dpmac";
 		};
+
 		dpmac@12 {
+			compatible = "fsl,dpmac";
 		};
+
 		dpmac@13 {
+			compatible = "fsl,dpmac";
 		};
+
 		dpmac@14 {
+			compatible = "fsl,dpmac";
 		};
+
 		dpmac@15 {
+			compatible = "fsl,dpmac";
 		};
+
 		dpmac@16 {
+			compatible = "fsl,dpmac";
 		};
+
 		dpmac@17 {
+			compatible = "fsl,dpmac";
 		};
+
 		dpmac@18 {
+			compatible = "fsl,dpmac";
 		};
+
 		dpmcp@1 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@2 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@3 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@4 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@5 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@6 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@7 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@8 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@9 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@10 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@11 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@12 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@13 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@14 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@15 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@16 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@17 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@18 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@19 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@20 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@21 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@22 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@23 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@24 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@25 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@26 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@27 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@28 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@29 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@30 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@31 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@32 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@33 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@34 {
+			compatible = "fsl,dpmcp";
 		};
 
 		dpmcp@35 {
+			compatible = "fsl,dpmcp";
 		};
 	};
 
-- 
2.35.3

